control/controlclient/map.go

```diff
@@ -319,6 +320,9 @@ func (ms *mapSession) updateStateFromResponse(resp *tailcfg.MapResponse) {
 	if resp.TKAInfo != nil {
 		ms.lastTKAInfo = resp.TKAInfo
 	}
+	if resp.MaxKeyDuration != 0 {
```

Comment:
Not that it's likely, but probably `resp.MaxKeyDuration > 0` ? (Duration is signed)